### DISCUSSION TITLE
Code Review TODO

### DISCUSSION THREAD

**Discussion by kbanker** [deepskies/DeepSZSim#118] [General]
- [x]  Add units to function definitions and other variables (arguments specifically)
- [ ] Top level directory should be cleaner (move physics to inner dir)
- [x] Fix read_yaml and load_vars (load_vars initialized multiple times)
- [ ] Clean up unused functions in notebook + modules
- [x] Standardize get vs make (in get_dm_halo vs make_sz_cluster): leaning towards renaming to make
- [ ] Make the start/end of the different parts of the notebook clearer (Making distribution vs Running sims using them)
- [x] Update reading in massdist from csv to h5
- [x] Rename M200_to_R200 function to reflect use
- [x] Create universal Eq11 from Battaglia2012, then move the param calculations into their own functions
-  [x] make x-axis log scale for Compton profile 1-d plot
-  [x] Make separate toolbox module for noise/beam convolution/fSZ calculation 
- [x] Look into CMB generation assumed geometry from pixell
- [ ] Halo ID for each halo
- [ ] Run ID for the sim run
- [x] Tool/filter module for aperture photometry
- [x] Move noise method from run_sims to a noise module

**Comment by evavagiakis** [comment]
Mostly completed, moving into two discussions: https://github.com/deepskies/SimSZ/discussions/153 for packaging and https://github.com/deepskies/SimSZ/discussions/154 for physics to dos
